Gemfile

```diff
@@ -99,6 +99,8 @@ gem 'riiif', '~> 1.1'
 
 gem 'iiif_manifest', '~> 0.1.2'
 
+gem 'carrierwave-aws'
```

Comment:
I'm trying to test with it in, since I hope for it to resolve a concrete problem.  Maybe that is not worth the effort though.